packages/travix-ui-kit/components/stepper/stepper.md

```diff
@@ -0,0 +1,35 @@
+Basic input:
+
+    <div style={{ background: '#D5E8F8', padding: '10px' }}>
+      <style dangerouslySetInnerHTML={{__html: `
+        .ui-stepper .ui-stepper-button.ui-stepper-button_decrease:before {
+          content: '-';
+        }
+        .ui-stepper .ui-stepper-button.ui-stepper-button_increase:before {
+          content: '+';
+        }
+      `}} />
+      <div style={{ width: '50%' }}>
```

Comment:
You don't need these `style` attributes, just the `content` overrides; the default look is fine.